base/libgit2/error.jl

```diff
@@ -96,7 +99,7 @@ macro check(git_func)
         local err::Cint
         err = $(esc(git_func::Expr))
         if err < 0
-            throw(Error.GitError(err))
+            throw(Error.GitError(err; path = gitdir($(esc(:repo)))))
```

Comment:
there are a few places `@check` is used in functions that don't have a `repo` argument, like some of the `GitConfig` constructors